cunumeric/array.py

```diff
@@ -1574,8 +1571,7 @@ def argmax(self, axis=None, out=None):
             self,
             axis=axis,
             dtype=np.dtype(np.int64),
-            dst=out,
-            check_types=False,
+            out=out,
         )
```

Comment:
Can you add the missing `@add_boilerplate` on argmax and argmin?

Reply:
0983a85268d3ec45a7907e04bd342e4c87611909